Tomato App/ViewController.swift

```diff
@@ -1,19 +1,211 @@
-//
-//  ViewController.swift
-//  Tomato App
-//
-//  Created by Dinmukhammed Begaly on 24.10.2023.
-//
-
 import UIKit
+import SnapKit
 
+var isWorkTime = true
+var isAnimationStarted = false
 class ViewController: UIViewController {
 
+
+    //MARK: - UI elements
+    private let foreProgresBar = CAShapeLayer()
+    private let backProgresBar = CAShapeLayer()
```

Comment:
Better to not shorten variable/constants name, foreground/background is more understandable for someone reading your code